### DISCUSSION TITLE
Entry Point of the official Marlin firmware for Kobra Go

### DISCUSSION THREAD

**Discussion by xiaopeng12138** [1coderookie/KobraGoNeoInsights#8] [General]
This is the entry point of the hc32f460
https://github.com/ANYCUBIC-3D/Kobra_Go/blob/e675180de25a622936be74ddf364f59715ec3943/source/app/main.cpp

Most custom stuffs are in the source/board folder. Maybe someone can find more interesting things.
hc32f460 is a quiet powerful mcu, add some additional function should not be hard

**Comment by 1coderookie** [comment]
Thanks!
